packages/victory/src/index.d.ts

```diff
@@ -3,19 +3,20 @@
 
 declare module "victory" {
   import {
+    Arc,
     Background,
-    // Border,
-    // Box,
-    // ClipPath,
-    // LineSegment,
-    // Whisker,
-    // Circle,
-    // Rect,
-    // Line,
+    Border,
+    Border as Box,
+    ClipPath,
+    LineSegment,
+    Whisker,
+    Circle,
+    Rect,
+    Line,
     // Path,
```

Comment:
Is there a reason `Path` is skipped?

Reply:
Nope! Good find, I was conflating `Path` with `PathHelpers` in my my mind, and was going to give you another PR with all the helper methods exported (brace yourself). Thanks for that catch! Adding now